styles.css

```diff
@@ -0,0 +1,963 @@
+/*Fonts*/
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Dancing+Script:wght@500&display=swap");
```

Comment:
Don't use more than 2 fonts for your app , i think lesser the font you use your app's performance will be optimized

Reply:
Actually I only use one font all over library. Other 2 fonts are just for my brand logo.
So I am avoiding image there.